docs/source/ingest/destination_connectors/vectara.rst

```diff
@@ -0,0 +1,32 @@
+Vectara
+===========
+
+Process all your records using ``unstructured-ingest`` to store structured outputs locally on your filesystem and upload those to a Vectara corpus.
+
+First you'll need to install the Vectara dependencies as shown here.
```

Comment:
Am I correct that there are no Vectara dependencies? Is there a Vectara python package that is used? If not you can delete these couple of lines.

Reply:
yes no dependencies. It's purely an API, so will remove this from the doc.